rholang/src/main/scala/coop/rchain/rholang/interpreter/Registry.scala

```diff
@@ -977,7 +977,7 @@ class RegistryImpl[F[_]](
           val Some(Expr(GInt(_)))              = nonce.singleExpr
           // Then check the signature
           val Some(Expr(GByteArray(sigBytes))) = sig.singleExpr
-          if (keyBytes.size == Validator.Length && sigBytes.size == 71 &&
+          if (keyBytes.size == Validator.Length &&
```

Comment:
Why not check if it is either 70 or 71?

Reply:
It doesn't matter as the verify call will just fail if it has the wrong size